packages/net/http/mimetypes.pony

```diff
@@ -0,0 +1,36 @@
+primitive MimeTypes
+  """
+  Provide mapping from file names to MIME types.
+  TODO load from /etc/mime.types
+  """
+  fun apply( name: String ): String val^ =>
+    """
+    Mapping is based on the file type, following the last period in the name.
+    """
+    try
+      // This will fail if no period is found.
+      let dotpos = (name.rfind( ".", -1, 0 )+1).usize()
```

Comment:
Standard library formatting would have no spaces between the arguments and the parentheses. Spaces should also be added around the addition so that it looks like `(name.rfind(".", -1, 0) + 1).usize()`